[32m2024-04-04 20:11:25 -0500[0m - dagster - [34mDEBUG[0m - __ASSET_JOB - a7bf5862-20a6-4d8e-a79d-b16c906140d9 - 45598 - LOGS_CAPTURED - Started capturing logs in process (pid: 45598).
[32m2024-04-04 20:11:25 -0500[0m - dagster - [34mDEBUG[0m - __ASSET_JOB - a7bf5862-20a6-4d8e-a79d-b16c906140d9 - 45598 - ncaa_rankings - STEP_START - Started execution of step "ncaa_rankings".
[32m2024-04-04 20:11:25 -0500[0m - dagster - [34mERROR[0m - [31m__ASSET_JOB - a7bf5862-20a6-4d8e-a79d-b16c906140d9 - 45598 - ncaa_rankings - STEP_FAILURE - Execution of step "ncaa_rankings" failed.

dagster._core.errors.DagsterExecutionStepExecutionError: Error occurred while executing op "ncaa_rankings"::

FileNotFoundError: [Errno 2] No such file or directory: '..data'

Stack Trace:
  File "/Users/isaacanderson/anaconda3/envs/boo/lib/python3.12/site-packages/dagster/_core/execution/plan/utils.py", line 54, in op_execution_error_boundary
    yield
  File "/Users/isaacanderson/anaconda3/envs/boo/lib/python3.12/site-packages/dagster/_utils/__init__.py", line 463, in iterate_with_context
    next_output = next(iterator)
                  ^^^^^^^^^^^^^^
  File "/Users/isaacanderson/anaconda3/envs/boo/lib/python3.12/site-packages/dagster/_core/execution/plan/compute_generator.py", line 131, in _coerce_op_compute_fn_to_iterator
    result = invoke_compute_fn(
             ^^^^^^^^^^^^^^^^^^
  File "/Users/isaacanderson/anaconda3/envs/boo/lib/python3.12/site-packages/dagster/_core/execution/plan/compute_generator.py", line 125, in invoke_compute_fn
    return fn(context, **args_to_pass) if context_arg_provided else fn(**args_to_pass)
                                                                    ^^^^^^^^^^^^^^^^^^
  File "/Users/isaacanderson/Desktop/CSE314/MarchMadness/projectDag/projectDag/assets/ETL/__init__.py", line 12, in ncaa_rankings
    df = c.read_to_one_frame('..data')
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/isaacanderson/anaconda3/envs/boo/lib/python3.12/site-packages/ncaa_mod/cleaning.py", line 9, in read_to_one_frame
    for file in os.listdir(path):
                ^^^^^^^^^^^^^^^^
[0m
